spec/support/helpers.rb

```diff
@@ -88,11 +90,11 @@ def load_error_run(ruby, name, *args)
     end
 
     def lib
-      File.expand_path("../../../lib", __FILE__)
+      File.expand_path("#{Spec::Path.root}/lib", __FILE__)
```

Comment:
since `Spec::Path` is included most everywhere `Helpers` is included, you should just be able to use `root`

additionally, please use `Pathname.join` instead of all the string interpolation and the `File.expand_paths`